packages/adapter-vercel/index.js

```diff
@@ -189,7 +189,8 @@ export default function ({ external = [], edge, split } = {}) {
 					platform: 'browser',
 					format: 'esm',
 					external,
-					sourcemap: 'linked'
+					sourcemap: 'linked',
+					banner: { js: 'var global = globalThis;' }
```

Comment:
this is more future-proof

```suggestion
					banner: { js: 'globalThis.global = globalThis;' }
```